### ISSUE TITLE
[BUG]: Improve responsiveness of landing page

### ISSUE THREAD

**Issue by ShreyaPrasad1209** [blossomlabsio/bloom#41]
### Please describe the bug

<img width="506" alt="image" src="https://github.com/blossomlabsio/bloom/assets/43600306/d9e489f2-e99b-43ad-9aeb-8f1098684596">

### Describe the device where you encountered the bug on

MacOS Sonoma, Chrome

**Comment by jatinkrmalik** [comment]
@ShreyaPrasad1209 Is there an issue from backend? Can you describe further as I didn't get it from the screenshot? 

Edit: are you talking about CSS responsiveness? To render the UI better?

**Comment by ShreyaPrasad1209** [comment]
> @ShreyaPrasad1209 Is there an issue from backend? Can you describe further as I didn't get it from the screenshot?
> 
> Edit: are you talking about CSS responsiveness? To render the UI better?

@jatinkrmalik yes I'm talking about CSS responsiveness. This is not related to backend.
